drivers/src/sha384.rs

```diff
@@ -61,7 +59,11 @@ impl Sha384 {
     ///
     /// * `data` - Data to used to update the digest
     ///
-    #[cfg_attr(not(feature = "no-cfi"), cfi_impl_fn)]
+    #[cfg_attr(
+        not(feature = "no-cfi"),
+        cfg_attr(feature = "runtime", caliptra_cfi_derive_git::cfi_impl_fn),
```

Comment:
Can you try something like this in the root module?

```
#[cfg(feature = "runtime")]
use caliptra_cfi_derive_submodule as caliptra_cfi_derive;

#[cfg(not(feature = "runtime"))]
use caliptra_cfi_derive_internal as caliptra_cfi_derive;
```

Reply:
Done